Framework/MDAlgorithms/test/Integrate3DEventsTest.h

```diff
@@ -39,8 +39,10 @@ class Integrate3DEventsTest : public CxxTest::TestSuite {
     V3D peak_1(10, 0, 0);
```

Comment:
Is it possible to add some test assertions that the errors are now as expected. This will help avoid future regressions in the code.